courses/machine_learning/deepdive2/recommendation_systems/solutions/content_based_by_hand.ipynb

```diff
@@ -0,0 +1,525 @@
+{
```

Comment:
<p>Not needed anymore with 2.0.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/691/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Some people run these locally on their own computers instead of on the cloud. I'd prefer to keep it as platform agnostic as possible.

Reply:
<p>Ack</p>